src/V1/FabricationFile/Actor.php

```diff
@@ -163,6 +164,31 @@ public function setTemplateRelativeFilePath(string $TemplateRelativeFilePath): A
         return $this;
     }
 
+    public function getPreferredTemplateTrees()
```

Comment:
```suggestion
    public function getPreferredTemplateTrees(): array
```

Reply:
*doh* Thanks. I forgot about the return types.